build.gradle.kts

```diff
@@ -34,12 +35,9 @@ detekt {
 }
 
 repositories {
-    jcenter()
+    mavenCentral()
```

Comment:
Just a reminder: org.jetbrains.kotlinx:kotlinx-html-jvm:0.7.2 is not yet on mavenCentral